### ISSUE TITLE
native vs non-native app: should we use ionic?

### ISSUE THREAD

**Issue by didac3141592** [Singulars2021/tender#6]
## Is Ionic worth-it?

TLDR: i think it is

So basically I watched [this tutorial](https://academind.com/tutorials/ionic-vue-first-app/) and then played a little bit with a project from scratch using the [official docs](https://ionicframework.com/docs) and [this is what I was able to do in a couple of hours](https://drive.google.com/file/d/1istAqedCYjB1rwoT5kMldeTC4QYMTvyE/view?usp=sharing).

From my POV, using Ionic we get: 

1. Easy Vue-3 implementation (works just like a Vue project with the addition of Ionic components)
2. A lot of components we could take advantatge of: 
-IonAlert
-IonBadge
-IonCard
-IIonSlides
-IonChip
-IonAvatar
-etc
3. Default navigation animations
4. Components can be stylized just like any Vue component
5. Dark Mode option + gets desired theme by device settings

To that, I should add that Ionic uses Capacitor to deploy both a Material-Design based Android app and a iOS-whatever based design for iOS (this is done automatically so no need to worry about it). 

Plus: Capacitor offers interesting pluggins we could use to get geolocalization, camera and local storage/gallery access.

However, I should add that I found the Ionic CLI error-handling process too permisive and that had some trouble with the IonIcon component (basically was not showing). I'm also afraid it could be a pain in the ass to customize some things, even though it is apparently fully customizable using either HTML tags, classes or ids. I also missed a more exhaustive documentation when it comes to its components. 

In conclusion,  I don't think using Ionic will result in a bigger workload.
